Docs/source/usage/parameters.rst

```diff
@@ -1452,6 +1452,14 @@ Grid initialization
     the field solver. In particular, do not use any other boundary condition
     than periodic.
 
+* ``warpx.maxlevel_extEMfield_init`` (default is maximum number of levels in the simulation)
+    With this parameter, the externally applied electric and magnetic fields
+    will not be applied for levels greater than ``warpx.maxlevel_extEMfield_init``.
+    For some mesh-refinement simulations,
+    the external fields are only applied to the parent grid and not the refined patches. In such cases,
+    ``warpx.maxlevel_extEMfield_init`` can be set to 0. Note that the other levels will be initialized to
+    a default value of 0.
```

Comment:
"Note that the other levels will be initialized to a default value of 0."

I am not sure that is true of we do an initial poisson solve.
Better would be:
```suggestion
    ``warpx.maxlevel_extEMfield_init`` can be set to 0.
    In that case, the other levels will not have external fields applied.
```